lib/moon/components/table/paging.ex

```diff
@@ -13,7 +13,7 @@ defmodule Moon.Components.Table.Paging do
 
   def render(assigns) do
     ~F"""
-    <div class="w-full flex">
+    <div class="w-full flex" data-testid="paging">
```

Comment:
:neckbeard: 
Perhaps passing the `data-test-id` attribute would work better due to namespace conflicts when rendering multiple paged tables.

Reply:
Actually, after adding it, I started thinking (never too late).
Paging is not used directly, its used implicitly by Moon.Components.Table. Which means its useless to have testid prop for Paging since I can't use it outside.
Also is this a problem, given that inside the same Paging component there are "next-page" and "prev-page" spans with static test ids?
@phgrey @lauaviin